ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/ExperimentalRuleSetProvider.kt

```diff
@@ -3,7 +3,7 @@ package com.pinterest.ktlint.ruleset.experimental
 import com.pinterest.ktlint.core.RuleSet
 import com.pinterest.ktlint.core.RuleSetProvider
 
-class ExperimentalRuleSetProvider : RuleSetProvider {
+public class ExperimentalRuleSetProvider : RuleSetProvider {
```

Comment:
Was this change necessary?

Reply:
Linter was complaining in the IDE that it needed an explicit `public` declaration